src/main/java/org/fundacion/jala/converter/view/converter/ResolutionVideo.java

```diff
@@ -38,14 +44,16 @@ public void setWidth(final String newWidth) {
 
     /**
      * Gets height of video.
-     * @return height of video.
+     *
+     * @return a height of video.
```

Comment:
would be better to put:
return video's height. 
Both on the method's name and the return.